script/Arcueid/Day 5/05_00_ARC05_3.txt

```diff
@@ -26,12 +26,12 @@ And besides, how are we going to find him in the first place?"
 [sha:c9a6d685b339c21acda59c88beb831f3a218cadd]{
 -- Page 219, Offset 9137.
 -- 「見つけるって……志貴ってほんと極端なのね。逃げないって決めたら、今度はこっちから攻めに行こうだなんて」
-"Find him... you really are extreme, aren't you? To think that you would go on the offensive right after choosing not to run away."
+"Find him... you really are extreme, aren't you? You've only just chosen to stand your ground, and now you want to go on the offensive."
 }
 [sha:d8cb65cc1a24079338e5be4b4f5a0cffc5768445]{
 -- Page 219, Offset 9138.
 -- 「な、なんとなくそう思っただけだっ！　極端の化身みたいなおまえに言われたくないっ」
-"H-Hey, I was just thinking out loud! I don't want to hear that from the personification of extremes herself."
+"H-Hey, I was just thinking out loud! I don't want to hear that from the queen of extremity herself."
```

Comment:
This line is fine, I just shudder to think that powerscalers will abuse this and say that arc has powers over extremes.